_i18n/nb-no/resources/user-guides/howto_fix_stuck_funds.md

```diff
@@ -0,0 +1,32 @@
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+
+Av og til vil midlene dine bli «sittende fast» – du vil ha noen låste midler som aldri blir ulåst. Slik kan du fikse det:
+
+- Last inn lommeboken din i monero-wallet-cli.
+
+- Tast inn
+
+> seed
+
+i ledeteksten. Skriv ned frøet ditt på 25 ord dersom du ikke allerede har gjort det. Dette er den beste måten å sørge for at du ikke mister tilgang til midlene dine på.
+
+- Lukk monero-wallet-cli ved å taste inn
+
+> exit
+
+- Sikkerhetskopier alle dine lommeboksrelaterte filer. Disse inkluderer:
```

Comment:
change to: "Sikkerhetskopier alle de lommebokrelaterte filene dine.

Reply:
Fixed